phpstan.neon

```diff
@@ -0,0 +1,9 @@
+parameters:
```

Comment:
The previous and next files being removed,  `config/.gitkeep` and `web/modules/custom/.gitkeep`.  These files are there to keep the structure until config is export, and a custom module is added.

Reply:
i added back the config directory but honestly, I don't think we should keep the custom directory until we actually need it because what if we never use it. granted it won't do any harm if it is there. Would you like to see it back @thejimbirch

Reply:
No, I haven't but ok. I just added it back.